frontend/src/app/modules/availability/new-slot/questions/questions.component.sass

```diff
@@ -76,3 +118,21 @@ $question-input-width: 337px
     cursor: ns-resize
     .cdk-drag:not(.cdk-drag-placeholder)
         transition: transform 250ms cubic-bezier(0, 0, 0.2, 1)
+
+.checkbox-container
+    display: flex
+    flex-direction: row
+    align-items: center
+    margin-top: 10px
+    margin-bottom: 10px
```

Comment:
These margins are redundant because parent container have `align-items: center`, thus the items are centered vertically